bundle.Dockerfile

```diff
@@ -6,7 +6,7 @@ LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=mcg-osd-deployer
 LABEL operators.operatorframework.io.bundle.channels.v1=alpha
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.18.1
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.18.0+git
```

Comment:
Why do we need to downgrade the operator sdk version?

Reply:
will update the version to latest version